pkg/utils/tekton/controller.go

```diff
@@ -556,6 +556,22 @@ func (s *SuiteController) CreateEnterpriseContractPolicy(name, namespace string,
 	return ec, s.K8sClient.KubeRest().Create(context.TODO(), ec)
 }
 
+// GetEnterpriseContractPolicy in a namespace
+func (k KubeController) GetEnterpriseContractPolicy(name, namespace string) (ecp.EnterpriseContractPolicy, error) {
```

Comment:
should there be a pointer to a struct returned instead?

Reply:
@psturc PTAL for the new change,thx